src/Microsoft.Fast.Components.FluentUI/Components/Progress/FluentProgress.razor.cs

```diff
@@ -1,9 +1,18 @@
 using Microsoft.AspNetCore.Components;
+using Microsoft.Fast.Components.FluentUI.Utilities;
 
 namespace Microsoft.Fast.Components.FluentUI;
 
 public partial class FluentProgress : FluentComponentBase
 {
+    protected string? ClassValue => new CssBuilder(Class)
+        .Build();
+
+    protected string? StyleValue => new StyleBuilder()
+        .AddStyle(Style)
+        .AddStyle("visibility", "hidden", () => Visible == false)
```

Comment:
Using `visibility: hidden`, the space used by the component will be there. You have to use `display: none;`.

But it's better to use a `@if (Visible) { ... }` in the razor file to remove the HTML code from the page when the component is invisible. The HTML code will be smaller.